### ISSUE TITLE
Documentation for Remote Container Extension Settings

### ISSUE THREAD

**Issue by BrutalSimplicity** [microsoft/vscode-remote-release#6128]
I don't see documentation for the available settings for the VSCode Remote extensions. Is this documentation maintained elsewhere?

I was looking for more information around the usage of the `remote.containers.dockerComposePath` setting, but was unable to find anything publicly documented. I do see that there is documentation in the extension settings page, but thought that these would be available in the online VSCode documentation.

Can we have the Remote extensions settings documentation made publicly available, or could you point me to where that documentation would be maintained (perhaps I can start this)?

**Comment by chrmarti** [comment]
It is either the name of an executable on the PATH or an absolute path to the executable. Documentation is sparse, Docker Path is mentioned here: https://code.visualstudio.com/docs/remote/containers#_can-i-use-podman-instead-of-docker.

@bamurtaugh Should we add a reference section for the Remote-Containers settings to the Advanced Containers Configuration documentation? Maybe we can improve the in-product wording and just copy that to the documentation page to make it searchable from the browser.

**Comment by bamurtaugh** [comment]
A centralized explanation of all Remote-Containers settings makes sense to me too - we can add a page to [Advanced Containers](https://code.visualstudio.com/remote/advancedcontainers/overview) like you mention, and it'd feel similar to how we strive to comprehensively document the [devcontainer.json properties](https://code.visualstudio.com/docs/remote/devcontainerjson-reference).

I'm happy to self-assign this and tackle this hopefully ~this iteration. I can start off by pulling the wording from in-product like you mention, and we can then tweak both the doc and actual in-product settings as necessary to be the most helpful to users as we go through this process.

**Comment by bamurtaugh** [comment]
I started on this, and I was wondering if I should pull the exact wording from the Settings that are documented under Feature Contributions on the extension overview page in-product?

![image](https://user-images.githubusercontent.com/25310137/148301761-f1bf6605-e0d6-4374-9f1a-cd3fc542e37c.png)

I want to double-check the value-add of a new docs page, as opposed to pointing users to the Feature Contributions section. Would it mainly help in being searchable from the browser if users have a question on a specific setting @chrmarti? Would it potentially make more sense to point users from docs to check out Feature Contributions in-product, so that if we change/add a setting, we don't need to remember to update it in two locations? 

I'm happy to add the doc, just wanted to discuss a bit further as I started working on it.
